Resources/Storage/StorageElement.py

```diff
@@ -131,7 +131,7 @@ def __init__( self, name, plugins = None, vo = None ):
         return
       self.vo = result['Value']
     self.opHelper = Operations( vo = self.vo )
-    
+
```

Comment:
This might actually be me, but I checked my eclipse settings, and they are like the dirac site recommends. Which setting do you have, so that it trims empty lines ?